parse_and_import_medicover.py

```
@@ -51,6 +51,7 @@ def main(
     write: bool,
     db_import: bool,
     dump: str = None,
+    mapping_refAlt: str = None,
```

Ruff (0.15.2)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)